src/components/teasers/teaser.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Teaser } from './teaser';
+import ExampleImage from '../../../blog-posts/images/angular.png';
+import styled from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { ILLUSTRATION_NAMES } from '../illustration/illustration-set';
+
+export default {
+  component: Teaser,
+  title: 'Components/Teaser',
+} as ComponentMeta<typeof Teaser>;
+
+const Template: ComponentStory<typeof Teaser> = (args) => <Teaser {...args} />;
+
+export const Regular = Template.bind({});
+Regular.args = {
+  title: 'Leadbox Title',
+  children:
+    'Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam eget risus varius blandit sit amet non magna.',
+  linkTo: '/blog',
+};
+
+const icons = {};
+{
+  ILLUSTRATION_NAMES.forEach((name) => {
+    icons[name] = (
+      <Illustration key={name} show={name} size={IllustrationSize.MEDIUM} />
+    );
+  });
+}
+
+export const WithIllustration = Template.bind({});
+WithIllustration.args = {
+  ...Regular.args,
+  cover: (
+    <Illustration show={'space_shuttle_043'} size={IllustrationSize.MEDIUM} />
```

Comment:
That feels okay. Either pass in an image or an illustration. Now seeing this I can imagine two variants:

```
<Teaser illustration='keyword'/>
<Teaser cover={<img/> }/>
```

where both props are mutually exclusive or one wins over the other. Something for a later refactoring if peopel agree.